### PR TITLE
[WIP] PBI-6: Users can view and share invitation for class code

### PR DESCRIPTION
**Pull request by darren-ngoh**

### CODE REVIEW

File: src/BadasoLMSModule.php [hunk 1]
```diff
@@ -2,12 +2,12 @@
 
 namespace Uasoft\Badaso\Module\LMS;
 
-class BadasoLMSModule {
```
Comment (IlmaMannix) [inline src/BadasoLMSModule.php:5]: do not commit this again, please check previously existing file on the staging branch

File: src/Commands/BadasoLMSSetup.php [hunk 2]
```diff
@@ -35,23 +37,36 @@ public function __construct() {
...
-    protected function addingBadasoEnv() {}
+    protected function addingBadasoEnv()
+    {
+    }
 
-    protected function addPostTablesToHiddenTables() {}
+    protected function addPostTablesToHiddenTables()
+    {
+    }
 }
```
Comment (IlmaMannix) [inline src/Commands/BadasoLMSSetup.php:72]: this too, please pull the latest version on the staging branch

File: tests/Feature/viewAndShareClassCode.php [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+namespace Uasoft\Badaso\Module\LMS\Tests\Feature;
+use Tests\TestCase;
+use Uasoft\Badaso\Helpers\CallHelperTest;
+
+class ViewAndShareClassCodeTest extends TestCase {
+    public function testClassCodeExist() {
+        // init user login as normal user
+        CallHelperTest::handleUserAdminAuthorize($this);
+    }
+}
```
Comment (IlmaMannix) [inline tests/Feature/viewAndShareClassCode.php:12]: please add new line

**Review by fxdros — COMMENTED** [review]
Please kindly check your StyleCI ☺️ ☺️

**Pull request closed without merging**